server/src/internalClusterTest/java/org/elasticsearch/get/GetActionIT.java

```diff
@@ -818,6 +845,75 @@ void indexSingleDocumentWithStringFieldsGeneratedFromText(boolean stored, boolea
         index("test", "1", doc);
     }
 
+    public void testAvoidWrappingSearcherInMultiGet() {
+        SearcherWrapperPlugin.enabled = true;
+        assertAcked(
+            prepareCreate("test").setMapping("f", "type=keyword")
+                .setSettings(
+                    Settings.builder()
+                        .put("index.refresh_interval", "1h")
```

Comment:
why not -1 here? That would seem more natural, avoiding to think about whether there could be a first scheduled refresh involved.

Reply:
I fixed this in https://github.com/elastic/elasticsearch/pull/104227/commits/5dd8b04dd66f32c750a66c2c564ad9a23891e6d1.